config/conf.json

```diff
@@ -22,7 +22,8 @@
                 "generate_docs": true,
                 "include_in_ui": true,
                 "build": true,
-                "include_in_custom_dataproc": true
+                "include_in_custom_dataproc": true,
+                "include_in_custom_gce": true
```

Comment:
where will this flag be used?

Reply:
It will be used by the new Jenkinsfile used to generate custom GCE images (in addition to Dataproc). See the [companion Leonardo PR](https://github.com/DataBiosphere/leonardo/pull/1306) for details.